frontend/src/views/v-authorship.vue

```diff
@@ -119,21 +127,27 @@
                 font-awesome-icon.button(icon="user-edit")
                 span.tooltip-text Click to view the blame view of file
           span.fileTypeLabel(
-            v-if='!file.isBinary',
+            v-if='!file.isBinary && !file.isIgnored',
             v-bind:style="{\
               'background-color': fileTypeColors[file.fileType],\
               'color': getFontColor(fileTypeColors[file.fileType])\
               }"
           ) {{ file.fileType }}&nbsp;{{ file.lineCount }}
             |&nbsp;({{ file.lineCount - file.blankLineCount }})
           span.fileTypeLabel.binary(v-if='file.isBinary') binary&nbsp;
-        pre.hljs.file-content(v-if="file.wasCodeLoaded && !file.isBinary", v-show="file.active")
+          span.ignored-tag.fileTypeLabel(
+            v-if='file.isIgnored'
+          ) ignored ({{ file.lineCount }})&nbsp;
+        pre.hljs.file-content(v-if="file.wasCodeLoaded && !file.isBinary && !file.isIgnored", v-show="file.active")
```

Comment:
A little nitpicky, but would it be better to reorder the `v-if`s as:
`v-if="file.isBinary"`
`v-else-if="file.isIgnored"`
`v-else-if="file.wasCodeLoaded"`
So that you don't have to repeat `!file.isBinary && !file.isIgnored`?

Reply:
Implemented this suggestion.